java-frontend/src/test/java/org/sonar/java/UCFGJavaVisitorTest.java

```diff
@@ -245,6 +247,24 @@ public void implicit_usage_of_this() {
         "}", expectedUCFG);
   }
 
+  @Test
+  public void explicit_static_method_call() {
```

Comment:
I tend to think that testing with something else than "java.lang.String" could be better. You can define your own static method or use one of java.util.Objects maybe. Especially given the modification of the other test.